packages/cozy-harvest-lib/src/components/KonnectorModal.jsx

```diff
@@ -208,117 +224,95 @@ export class KonnectorModal extends PureComponent {
-                        className="u-mb-2"
-                        error={konnectorJobError}
-                        konnector={konnector}
-                      />
-                    )}
-                    <LaunchTriggerCard
-                      trigger={trigger}
-                      onError={this.handleKonnectorJobError}
-                      onLaunch={this.handleTriggerLaunch}
-                      onSuccess={this.handleKonnectorJobSuccess}
-                      submitting={isJobRunning}
-                    />
-                  </TabPanel>
-                  <TabPanel name="configuration" className="u-pt-1-half u-pb-2">
-                    {shouldDisplayError && hasLoginError && (
-                      <TriggerErrorInfo
-                        className="u-mb-2"
-                        error={konnectorJobError}
-                        konnector={konnector}
-                      />
-                    )}
-                    <div className="u-mb-1">
-                      <SubTitle className="u-mb-half">
-                        {t('modal.updateAccount.title')}
-                      </SubTitle>
-                      <TriggerManager
-                        account={account}
-                        konnector={konnector}
-                        trigger={trigger}
-                        onError={this.handleKonnectorJobError}
-                        onLaunch={this.handleTriggerLaunch}
-                        onSuccess={this.handleKonnectorJobSuccess}
-                        running={isJobRunning}
-                        showError={false}
-                      />
-                    </div>
-                    <div className="u-mb-2">
-                      <SubTitle className="u-mb-half">
-                        {t('modal.deleteAccount.title')}
-                      </SubTitle>
-                      <Text className="u-mb-1">
-                        {t('modal.deleteAccount.description')}
-                      </Text>
-                      <DeleteAccountButton
-                        account={account}
-                        disabled={isJobRunning}
-                        onSuccess={dismissAction}
-                        extension="full"
-                      />
-                    </div>
-                    <div>
-                      <SubTitle className="u-mb-half">
-                        {t('modal.addAccount.title')}
-                      </SubTitle>
-                      <Button
-                        onClick={createAction}
-                        label={t('modal.addAccount.button')}
-                        extension="full"
-                        theme="secondary"
-                      />
-                    </div>
-                  </TabPanel>
-                </TabPanels>
-              </Tabs>
-            )}
-          </ModalContent>
-        )}
+        <ModalContent>{this.renderModalContent()}</ModalContent>
       </Modal>
     )
   }
+
+  renderModalContent() {
```

Comment:
I don't like all these `renderXXX` because it means that the component should render multiple things, but as we want to keep the render method as small as possible we create an other render method to do so... 

Maybe we should create a `<ModalContentSwitcher>` component ?